.github/stale.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request with the stale
+# label is closed. Set to false to disable. If disabled, issues still need to
+# be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
```

Comment:
Maybe we'd like a parallel comment here noting the bot un-stale'd it... but, I'm not even sure how you make the bot do that - I presume you send some sort of `\bot-unstale` request?

Reply:
The doc is basically non-existent for this project. However, I think the way it works is that the bot will apply the comment specified here if the label gets removed! No special commands required - just remove the label ;) I'm not convinced we need to have a comment for that since if the label is being removed, that implies someone is going to work on it and hopefully apply a comment saying "I'm all over this now" ;)